index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Home &mdash; Weather Report</title>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" href="styles/index.css" />
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+    </style>
+</head>
+
+<body>
+    <main>
+        <header>
+            <h1 id="page-header">Weather report for City:</h1>
+        </header>
+
+        <section id=main-screen>
+            <section id="display-section">
+                <section id="weather-garden">
+                    <h2>Weather Garden</h2>
+                    <div id="garden-content">
+                        <div id="sky">☁️ ☁️ ☁️ ☀️ ☁️ ☁️</div>
+                        <div id="landscape"></div>
+                    </div>
+                </section>
+            </section>
+
+            <section id="control-section">
+                <section id="temperature-control">
+                    <h2>Temperature</h2>
+                    <form class="temperature-content">
+                        <span id="increase-temp">🔼</span>
+                        <span id="current-temp"></span>
+                        <span id="decrease-temp">🔽</span>
+                    </form>
+                </section>
```

Comment:
Be careful using having too many ids for every single html element. It's better practice to take advantage of pseudo selectors and the parent-child relationships, like:

`#control-section section` or `#control-section form` these selectors now isolate the elements inside the parent element, so that extra ids aren't necessary